sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -18,29 +18,59 @@ import com.vaadin.ui.themes.ValoTheme
 */
 class ProjectOverviewView extends VerticalLayout{
 
+    private final ProjectOverviewViewModel viewModel
+    private final DownloadProjectController downloadProjectController
+
     private Label titleLabel
-    private ProjectOverviewViewModel viewModel
     private Grid<ProjectSummary> projectGrid
+    private TextArea manifestArea
 
     final static int MAX_CODE_COLUMN_WIDTH = 400
     final static int MAX_STATUS_COLUMN_WIDTH = 200
 
-    ProjectOverviewView(ProjectOverviewViewModel viewModel){
+    ProjectOverviewView(ProjectOverviewViewModel viewModel, DownloadProjectController downloadProjectController){
         this.viewModel = viewModel
+        this.downloadProjectController = downloadProjectController
 
         initLayout()
-        fillGrid()
     }
 
     private void initLayout(){
         titleLabel = new Label("Project Overview")
         titleLabel.addStyleName(ValoTheme.LABEL_LARGE)
+        setupProjects()
+        manifestArea = setupManifestContent()
+        this.addComponent(setupProjectSpecificButtons())
+        this.addComponents(titleLabel, projectGrid)
+        this.addComponent(manifestArea)
+    }
+
+    private void setupProjects(){
         projectGrid = new Grid<>()
+        fillProjectsGrid()
+        projectGrid.setSelectionMode(Grid.SelectionMode.SINGLE)
+        projectGrid.addSelectionListener({
+            if (it instanceof SingleSelectionEvent<ProjectSummary>) {
+                clearProjectSelection()
+                it.getSelectedItem().ifPresent(this::selectProject)
```

Comment:
What is selectProject?

Reply:
> I am missing the download here. Why do you not download a file to the user's system. When should the file be created?

The download will be handled in a separate PR